src/screens/ProfileScreen/elements/RenderItem.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import LinearGradient from 'react-native-linear-gradient';
-import {Dimensions, StyleSheet, View} from 'react-native';
+import {Dimensions, StatusBar, StyleSheet, TouchableOpacity, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `StatusBar` import has been added. Ensure that this is compatible with all target platforms, as `StatusBar.currentHeight` is only available on Android. If the app also targets iOS, you may need to handle this differently.
```diff
- import {Dimensions, StyleSheet, View} from 'react-native';
+ import {Dimensions, StatusBar, StyleSheet, TouchableOpacity, View} from 'react-native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->